docs/index.md

```diff
@@ -111,22 +111,26 @@ Quantum Mobile has been used, amongst others, in the following schools and cours
 - [AiiDA tutorials](https://aiida-tutorials.readthedocs.io) in Switzerland, Italy, Slovenia, China, Japan, India, Norway, ...
 
 ```{list-table}
-* - ![cineca](_static/user_logos/cineca.png)
-  - ![ethz](_static/user_logos/ethz.png)
-  - ![ictp](_static/user_logos/ictp.png)
-  - ![jsi](_static/user_logos/jsi.png)
-  - ![sintef](_static/user_logos/sintef.png)
-  - 
-  - 
+---
+header-rows: 0
```

Comment:
I tried playing around with the `widths` argument but it seems it doesn't work (?)

A bit ugly these tables but I guess there are limits to what one can do with sphinx...

Reply:
Well I didn't find an existing sub-figure-esque sphinx extension; so this is my current "quick-fix", but its on my TODO list to add a proper extension for it.
There's not too much you can't do with sphinx, just need the right the extensions 😄